gammapy/utils/scripts.py

```diff
@@ -63,7 +63,7 @@ def make_name(name=None):
         return name
 
 
-def make_path(path):
+def make_path(path, abs_path=False):
```

Comment:
Maybe the change is not required if one simply add `.resolve()` to the path returned by `make_path` when it is needed.